Completion/Unix/Command/_git

```diff
@@ -7130,12 +7130,27 @@ __git_deleted_files () {
 
 (( $+functions[__git_modified_files] )) ||
 __git_modified_files () {
-  __git_files --modified modified-files 'modified file' $*
+  __git_status_files
+}
```

Comment:
Given that the incumbent code uses `$*` and the new code doesn't, I don't see how this can possibly be correct.  For starters, I expect it breaks completion of ignored files after `git add -f <TAB>` (see `_git-add`).